### ISSUE TITLE
Insert new mail merge fields using sablon 

### ISSUE THREAD

**Issue by coezbek** [senny/sablon#175]
Is there a way to let Sablon insert mail merge fields into the word template?

This would be helpful to build the template in the first place.

I imagine the following workflow:

- Build context object in Ruby
- Create the template with a merge field called something like `allFields`
- Run Sablon once to have it generate new merge fields for the whole context and insert them in the position of `allFields`.

**Comment by stadelmanma** [comment]
Hey @coezbek the word templates are finicky at times since docx files aren’t really designed in a friendly way to interact with outside of MS word itself.

That being said I’m not quite sure I understand. Are you meaning to insert the all the fields in the doc so that one could copy and paste them around?

**Comment by coezbek** [comment]
Yes, I meant to insert all entries from the context as fields into an
intermediate template to then move them around to produce a final template.
